common/scala/src/main/resources/application.conf

```diff
@@ -12,3 +12,22 @@ akka.http {
         max-open-requests = 128
     }
 }
+
+akka.tracing {
+    #zipkin configuration
+    host = "localhost"
+    host = ${?TRACING_HOST} # Override default HOST by enviornment variable
+    port = 9410 # default
+    port = ${?TRACING_PORT} # Override default PORT by enviornment variable
+    sample-rate = 100 # by default trace every 100th request
+    sample-rate = ${?TRACING_SAMPLE_RATE} # Override default Sample_Rate by enviornment variable
+
+    enabled = false
+    enabled = ${?TRACING_ENABLED} # Override default Enabled by enviornment variable
+    max-spans-per-second = 100
```

Comment:
`max-spans-per-second` allows to discard any span sampling above specified limit. So within a given sample rate if there are too many spans being sampled this setting can be used to control that.